cmd/release-controller/http_candidate.go

```diff
@@ -106,12 +105,23 @@ func (c *Controller) httpReleaseCandidateList(w http.ResponseWriter, req *http.R
 	defer func() { glog.V(4).Infof("rendered in %s", time.Now().Sub(start)) }()
 	vars := mux.Vars(req)
 	releaseStreamName := vars["release"]
-	releaseCandidateList, err := c.findReleaseCandidates(releaseStreamName)
+	successPercent := 80.0
+	releaseCandidateList, err := c.findReleaseCandidates(successPercent, releaseStreamName)
 	if err != nil {
+		if err == errStreamNotFound {
+			http.Error(w, err.Error(), http.StatusNotFound)
+			return
+		}
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
 
+	if releaseCandidateList[releaseStreamName] == nil {
+		releaseCandidateList[releaseStreamName] = &ReleaseCandidateList{
+			Items: make([]*ReleaseCandidate, 0),
```

Comment:
You don't need to do this in Golang - the different between nil and empty slices is not important, and so you would just do `&ReleaseCandidateList{}` here